ci/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG ARG_TEST_IMAGE_NAME=hexpm/elixir
+ARG ARG_TEST_IMAGE_VERSION=1.14.5-erlang-25.3.2.6-ubuntu-jammy-20230126
+
+FROM ${ARG_TEST_IMAGE_NAME}:${ARG_TEST_IMAGE_VERSION}
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    TZ=Etc/UTC \
+    DISPLAY=:99.0 \
+    CHROMEDRIVER_VERSION='104.0.5112.79' \
+    NODE_MAJOR=18
+
+RUN apt update \
```

Comment:
Since you are using pipes in your RUN commands you want to setup proper flags to catch errors: 
```RUN set -euxo pipefail \```